text/0000-channel-participation-api-without-system-channel.md

```diff
@@ -0,0 +1,497 @@
+operation can break existing on-boarding code in unpredictable ways.
+
+We propose to deal with this risk by clearly documenting that this is not supported and highly discouraged.
+  
+  
+# Rationale and alternatives
+[alternatives]: #alternatives
+
+## Rationale
+The proposed design completely decouples application channels from the system channel and thus, breaks the scalability 
+barrier. Each channel can hence be deployed on a different set of orderers, achieving linear scalability in the number
+of channels. This design also solves the privacy problem inherent in a shared system channel, and simplifies 
+admin operations.
+
+## Alternative
+An alternative design is to keep the system channel running on small cluster, and drop the requirement of app-channel
+OSNs to be a member of that cluster. OSNs belonging to app-channels follow the system channel and fetch blocks like 
+a peer does.
+
+This solves the scalability of channels, but not the privacy problem. Some of the operational problems also remain, 
+e.g. slow on-boarding time of new OSNs existing channels.
+
+# Prior art
+[prior-art]: #prior-art
+
+N/A
+
+# Testing
+[testing]: #testing
+
+## Unit tests
+Every new class of the new participation API will be covered by new unit tests.
+
+## Integration tests
+Integration tests will cover the following scenarios:
+1. Bootstrap: 
+   - starting a set of empty OSNs 
+   - joining all to an app-channel
+2. Add new channel: 
+   - starting with a set of OSNs with a channel 
+   - add another channel to a subset of OSNs (Join with genesis-block)
+3. Add new OSN: 
+   - start with a set of OSNs, with a channel, with blocks
+   - add the new OSN to the channel (join with last-config-block)
+   - as a follower (not in consensus set), adding it after catch-up
+   - as a member (in consensus set)
+4. Remove an OSN from a channel:
+   - start with a set of OSNs, with a channel, with blocks
+   - add the new OSN to the channel 
+        - as a follower 
+        - as a member
+5. Bootstrap to system channel:
+   - starting a set of empty OSNs 
+   - joining all to a system-channel
+   - create a channel
+   - make sure Join and Delete (app-channel) are disabled
+5. Transition from system-channel to local participation:
+   - start from a network with a system channel and app channel
+   - remove system-channel
+   - add channel with participation API
+
+
+# Dependencies
+[dependencies]: #dependencies
+
+## Existing items
+These Jira items are precursors of this RFC:
+
+- https://jira.hyperledger.org/browse/FAB-15709
+- https://jira.hyperledger.org/browse/FAB-15710
+- https://jira.hyperledger.org/browse/FAB-15711
+- https://jira.hyperledger.org/browse/FAB-15712
+- https://jira.hyperledger.org/browse/FAB-15713
+
+These will be combined in the development plan once the RFC is approved.
+
+## Authorization and Authentication
+There is an ongoing effort to improve the AA mechanisms of the "operations" API. 
```

Comment:
Where is that effort taking place? I'm very interested as it has larger impacts and is explicitly an area that I have opinions on.

Reply:
Gari is doing some exploratory work on that, I believe.